src/vs/workbench/parts/execution/electron-browser/terminal.ts

```diff
@@ -11,6 +11,8 @@ if (env.isLinux) {
 		defaultTerminalLinux = 'x-terminal-emulator';
 	} else if (process.env.DESKTOP_SESSION === 'gnome' || process.env.DESKTOP_SESSION === 'gnome-classic') {
 		defaultTerminalLinux = 'gnome-terminal';
+	} else if (process.env.DESKTOP_SESSION === 'kde-plasma') {
```

Comment:
@Tyriar hmm, yeah I was only really thinking about improving Kubuntu. This whole section may need work if it's going to include desktop-manager specific code, even just to handle the major flavors. 

That said, if old versions of KDE Fedora (or others) fall through the `if`, they're no worse off then they are now.

Another approach might be to have the workspace/user setting for the external terminal affect the debugger. When I tried setting it to `konsole`, there was no change in the "install gnome terminal" error message. Not sure if that's an issue in and of itself (or a feature) or what the side effects would be.